tests/testsuite/read_manifest.rs

```diff
@@ -116,6 +116,102 @@ fn simple_with_deps() {
     "reason": "manifest"
   }
 ]
+"#]]
+            .is_json()
+            .against_jsonlines(),
+        )
+        .with_status(0)
+        .run();
+
+    p.cargo_plumbing("plumbing read-manifest")
+        .arg("--manifest-path")
```

Comment:
Per usual, I ask that test changes that can be made in prior commits to do so (as previous iterations of this PR did).

I considered merging as-is but I figure having to go back and change this might pay off in helping to remember through the rest of this project :)